docs/product/base/grid.html

```diff
@@ -0,0 +1,725 @@
+            </div>
+            <div class="d-grid grid__4 mb16">
+                <div class="bg-black-075 p12 ba bc-black-3 grid--col-start2 grid--col-end4">.grid--col-start2 .grid--col-end4</div>
+            </div>
+            <div class="d-grid grid__4">
+                <div class="bg-black-075 p12 ba bc-black-3 grid--col2 grid--col-end5">.grid--col2 .grid--col-end5</div>
+            </div>
+        </div>
+    </div>
+
+    {% header h3 | Row start and end classes %}
+    <div class="overflow-x-auto mb48">
+        <table class="wmn4 s-table s-table__bx-simple">
+            <thead>
+                <tr>
+                    <th class="s-table--cell3" scope="col">Class</th>
+                    <th class="s-table--cell3" scope="col">Output</th>
+                    <th scope="col">Definition</th>
+                    <th class="s-table--cell2 ta-center" scope="col"><a class="s-link s-link__inherit" href="{{ "/product/guidelines/conditional-classes#responsive" | relative_url }}">Responsive?</a></th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for item in site.data.product.css-grid.row-start-end %}
+                    <tr class="fs-caption">
+                        <th scope="row"><code class="stacks-code">.{{ item.class }}</code></th>
+                        <td><code class="stacks-code bg-white">{{ item.output }}</code></td>
+                        <td>{{ item.define }}</td>
+                        <td class="ta-center">
+                            {% if item.responsive %}
+                                {% icon Checkmark | fc-green-500 %}
+                            {% endif %}
+                        </td>
+                    </tr>
+                {% endfor %}
+            </tbody>
+        </table>
+    </div>
+
+    {% header h3 | Row start and end examples %}
+    <div class="stacks-preview">
+{% highlight html %}
+<div class="d-grid grid__4">
+    <div class="grid--col2 grid--row-start2 grid--row-end4">…</div>
+    <div class="grid--col2">…</div>
+    <div class="grid--col2">…</div>
+    <div class="grid--col2">…</div>
+    <div class="grid--col2">…</div>
+</div>
+{% endhighlight %}
+        <div class="stacks-preview--example ff-mono ws-nowrap">
+            <div class="d-grid grid__4 gg16">
+                <div class="bg-black-075 p12 ba bc-black-3 grid--col2 grid--row-start2 grid--row-end4">
+                    <div>.grid--col2</div>
+                    <div>.grid--row-start2</div>
+                    <div>.grid--row-end4</div>
+                </div>
+                <div class="bg-black-075 p12 ba bc-black-3 grid--col2">.grid--col2</div>
+                <div class="bg-black-075 p12 ba bc-black-3 grid--col2">.grid--col2</div>
+                <div class="bg-black-075 p12 ba bc-black-3 grid--col2">.grid--col2</div>
+                <div class="bg-black-075 p12 ba bc-black-3 grid--col2">.grid--col2</div>
+            </div>
+        </div>
+    </div>
+</section>
```

Comment:
Additions I'd love to see:

- `justify-items` / `justify-self` / `justify-content`
- `align-items` / `align-self` / `align-content`
- `place-items` / `place-self` / `place-content`

Reply:
Skipping place-items since it's just shorthand. Adding `justify-items` and `justify-self`.